config/initializers/email_prefixer.rb

```diff
@@ -1,3 +1,3 @@
 EmailPrefixer.configure do |config|
-  config.stage_name = 'WHEEL'
+  config.stage_name = Rails.application.secrets.subject_prefix_for_outgoing_emails
```

Comment:
`EmailPrefixer` has two config options
1. `application_name`.
2. `stage_name`

I think in the above case, I think we should be configuring `application_name`.
`stage_name` is to be used for the environment name (if we want to give any custom env name).

The final prefix combination is of the form `[application_name stage_name]`. So to have something like `[WHEEL Development]` or `[WHEEL Staging]`, we'll have `application_name` as WHEEL and then we can optionally provide `stage_name` if required. I think we should stick to that convention. What do you think ?